### ISSUE TITLE
New YOLO models live in ultralytics/ultralytics 🚀

### ISSUE THREAD

**Issue by glenn-jocher** [ultralytics/yolov5#13796]
YOLOv5 remains available here for legacy projects, reproducibility, and users who specifically need the YOLOv5 codebase.

For all new Ultralytics models, current documentation, active development, and production workflows, use the main repository:

https://github.com/ultralytics/ultralytics

## Use the main repo for new projects

The `ultralytics/ultralytics` repository is the canonical home for the current Ultralytics YOLO family, including:

- YOLO26, the latest stable recommended model generation
- YOLO11 and YOLOv8 support
- Detection, segmentation, semantic segmentation, classification, pose, OBB, and tracking
- Current training, validation, prediction, export, and deployment workflows
- Latest docs, examples, bug fixes, and releases

## Install the current package

```bash
pip install -U ultralytics
```

## Run YOLO26 from the current package

```bash
# Predict
yolo predict model=yolo26n.pt source=https://ultralytics.com/images/bus.jpg imgsz=640

# Train
yolo train data=coco8.yaml model=yolo26n.pt epochs=10 imgsz=640

# Track
yolo track model=yolo26n.pt source=https://youtu.be/LNwODJXcvt4

# Export
yolo export model=yolo26n.pt format=onnx imgsz=640
```

## Python example

```python
from ultralytics import YOLO

model = YOLO("yolo26n.pt")
results = model("https://ultralytics.com/images/bus.jpg")
```

## Current docs and resources

- Main repository: https://github.com/ultralytics/ultralytics
- YOLO26 docs: https://docs.ultralytics.com/models/yolo26
- Python usage: https://docs.ultralytics.com/usage/python
- CLI usage: https://docs.ultralytics.com/usage/cli
- Train mode: https://docs.ultralytics.com/modes/train
- Predict mode: https://docs.ultralytics.com/modes/predict
- Export mode: https://docs.ultralytics.com/modes/export
- Tracking mode: https://docs.ultralytics.com/modes/track
- Roadmap: https://www.ultralytics.com/roadmap

## When should I still use this YOLOv5 repo?

Use this repository when you intentionally need YOLOv5 scripts, YOLOv5-era behavior, or old project compatibility.

For new models, new features, current docs, and active maintenance, start in `ultralytics/ultralytics`.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @glenn-jocher, thank you for the update on YOLOv5 and its legacy role 🚀 This is an automated response to help guide users arriving here, and an Ultralytics engineer will also assist soon 🤖
YOLOv5 remains a great option for legacy projects, reproducibility, and users who specifically need the YOLOv5 codebase. For anyone working in this repository, please visit our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5) to get started, where you can find quickstart guides for simple tasks like [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data) all the way to advanced concepts like [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution).
If you are opening a 🐛 Bug Report related to YOLOv5, please provide a **minimum reproducible example** to help us debug it.
If you are asking a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).
## Requirements
[**Python>=3.8.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). To get started:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install uv  # install uv
uv pip install -r requirements.txt  # install
```
## Environments
YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>
## Status
<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>
If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 [training](https://github.com/ultralytics/yolov5/blob/master/train.py), [validation](https://github.com/ultralytics/yolov5/blob/master/val.py), [inference](https://github.com/ultralytics/yolov5/blob/master/detect.py), [export](https://github.com/ultralytics/yolov5/blob/master/export.py) and [benchmarks](https://github.com/ultralytics/yolov5/blob/master/benchmarks.py) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by ChanifRusydi** [comment]
Hi, I know this might be a bit silly, but any recommendation or best practice on structuring a project that use both YOLOv5 (legacy) and the newer model (YOLOv8 and up) that uses ultralytics package (installed through pip), mainly to avoid confusion about import and modifying part of the model.
As mention, I'm currently using ultralytics package to call newer model, but recently consider to revisit older model as additional comparison and some recent research paper still using it
Any assistance would be greatly appreciated

**Comment by pderrenger** [comment]
For reproducible comparisons, keep two isolated runners instead of importing both into one Python process: pin a YOLOv5 fork/submodule to the paper’s commit in its own virtual environment, pin `ultralytics` in another, and invoke each through a thin CLI/subprocess adapter that produces the same prediction and metric format. Share only datasets and evaluation code, keep outputs/configs separate, and avoid adding the YOLOv5 root to global `PYTHONPATH`, since its top-level `models` and `utils` imports can cause confusion. For checkpoint-only comparisons the `ultralytics` API can load YOLOv5 weights, but use the pinned legacy repository when reproducing YOLOv5-era training behavior or modifying its architecture.

**Comment by ChanifRusydi** [comment]
I appreciate the insight and suggestion, will do as suggested, thank you

**Comment by Y-T-G** [comment]
@ChanifRusydi It's best to convert the YOLOv5 model to ONNX and rewire it using the ONNX modification code here (it works with YOLOv5 too) to make it work with Ultralytics:

https://docs.ultralytics.com/models/yolov7#onnx-export
